obsplus/structures/fetcher.py

```diff
@@ -199,7 +207,10 @@ def set_stations(self, stations: fetcher_station_type):
         except TypeError:
             self.station_client = getattr(self, "station_client", None)
         try:
-            self.station_df = stations_to_df(stations)
+            # since its common for inventories to have far out enddates this can
+            # raise a warning. These are safe to ignore.
```

Comment:
Is there any way to suppress the specific warning that you are referring to here? Just in case you add other warnings in the future? Or to add the option to surpess station end-date warnings in `stations_to_df`? Obviously adding another argument to `stations_to_df` isn't particularly desirable, but if you know that end-dates can be strange then it might be worth it?

Reply:
`stations_to_df` is an instance of `DataFrameExtractor`, not a function, so adding the ability to ignore targeted warnings would be rather difficult. Good point though, I have added a custom Warning which can be filtered specifically for this case.